pkg/shell/shell.scss

```diff
@@ -100,8 +100,23 @@ html.index-page body {
 .display-language-modal {
   /* Do not let the language menu expand to the end of the page */
   .pf-c-menu__content {
-    max-height: 20rem;
+    height: 11rem;
```

Comment:
Yeah, it shouldn't be that small. Also, setting a height on it will break mobile, unless special consideration is used.

It's probably best as `block-size: max(20rem, 100vh - var(--pf-global--spacer--3xl));` This would set the block size to 20rem or less, such as when the vertical height (100vh is 100% of the vertical viewport) and the bottom bar in mobile (which is the same height as spacer 3xl, which is 64px).